packages/navigation/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# 3.0.0 (unreleased)
+# 3.0.0
 
 - `getHistory` updated to reflect path parameters in url query.
```

Comment:
Could you add the following in here:

```
- Added a new `<Form />` component.
- Stepper component: Add new `content` and 	`description` props.
```

Reply:
Done in https://github.com/woocommerce/woocommerce-admin/pull/2802/commits/68f76aacad0de1dbebbed651f4984aaea63bfe3b